Common/src/main/resources/assets/hexcasting/lang/ru_ru.flatten.json5

```
@@ -1684,88 +1864,95 @@
+          "1": "Начертив $(l:patterns/patterns_as_iotas#hexcasting:open_paren)$(action)Интроспекцию/$, мои начертания рун начинают вести себя по-другому на какое-то время. Пока я не начерчу $(l:patterns/patterns_as_iotas#hexcasting:open_paren)$(action)Ретроспекцию/$, начертанные мною руны сохраняются в уме. Затем, когда я начерчу $(l:patterns/patterns_as_iotas#hexcasting:close_paren)$(action)Ретроспекцию/$, они добавляются на стек как список йот.",
+          "2": "Если я начерчу ещё одну $(l:patterns/patterns_as_iotas#hexcasting:close_paren)$(action)Интроспекцию/$, она все равно будет сохранена в списке, но затем мне придется начертить $(italic)две/$ $(l:patterns/patterns_as_iotas#hexcasting:close_paren)$(action)Ретроспекции/$, чтобы вернуться к обычному заклинанию.",
+        },
+        undo: "Наконец, если я допущу ошибку при начертании рун внутри $(l:patterns/patterns_as_iotas#hexcasting:open_paren)$(action)Интро-/$ и $(l:patterns/patterns_as_iotas#hexcasting:close_paren)$(action)Ретроспекций/$, можно начертить $(l:patterns/patterns_as_iotas#hexcasting:undo)$(action)Исчезновение/$, чтобы удалить последний начерченный мною узор из списка рун.",
+        further_notes: {
+          title: "Дополнительные записи",
+          "1": "я могу избежать специального поведения $(l:patterns/patterns_as_iotas#hexcasting:open_paren)$(action)Интро-/$ и $(l:patterns/patterns_as_iotas#hexcasting:close_paren)$(action)Ретроспекций/$, начертив $(l:patterns/patterns_as_iotas#hexcasting:escape)$(action)Рассмотрение/$ перед ними, что просто добавит их в список, не влияя на количество $(l:patterns/patterns_as_iotas#hexcasting:close_paren)$(action)Ретроспекций/$, необходимых для возвращения к заклинанию.$(br2)Если я начерчу две руны $(l:patterns/patterns_as_iotas#hexcasting:escape)$(action)Рассмотрения/$ подряд во время $(l:patterns/patterns_as_iotas#hexcasting:open_paren)$(action)Интроспекции/$, это добавит одно $(l:patterns/patterns_as_iotas#hexcasting:escape)$(action)Рассмотрение/$ в список.",
+          "2": "Если йота, не являющаяся руной, присутствует в списке на исполнение руной $(l:patterns/meta#hexcasting:eval)$(action)гамбита Гермеса/$ (или любой другой мета-руной исполнения), то произойдёт оишбка. Но я могу избежать этого, используя руны, описаные ниже.$(br2)Прямо как с йотами рун, руны ниже позволяют \"экранировать\" $(o)любые$() йоты, заставляя их пропихнуться на стек при исполнении вместо ошибки.",
+          "3": "Эта техника пригодится, если я захочу сослаться на специфичную йоту в своём $(#b38ef3)Заговоре/$, вроде сложного вектора или ссылки на сущность, без нужды создавать их с нуля при каждом исполнении.$(br2)Нужно не забыть о том, как именно я внесу такую йоту в список рун для исполнения. Простейшим способом будет сначала начертить "заглушку" при создании списка, а потом заменить её на нужную йоту с помощью руны $(l:patterns/lists#hexcasting:replace)$(action)Возвышения Хирурга/$.",
```

This should fix the build errors:
```suggestion
          "3": "Эта техника пригодится, если я захочу сослаться на специфичную йоту в своём $(#b38ef3)Заговоре/$, вроде сложного вектора или ссылки на сущность, без нужды создавать их с нуля при каждом исполнении.$(br2)Нужно не забыть о том, как именно я внесу такую йоту в список рун для исполнения. Простейшим способом будет сначала начертить \"заглушку\" при создании списка, а потом заменить её на нужную йоту с помощью руны $(l:patterns/lists#hexcasting:replace)$(action)Возвышения Хирурга/$.",
```